frontend/src/CreateAccount.js

```diff
@@ -0,0 +1,59 @@
+import React, {useState} from 'react';

```

Comment:
Is the CreateAccount file a copy of this file? Also refer to that file for comments

Reply:
@RandallCaler